messages/index.js

```diff
@@ -203,7 +203,7 @@ module.exports = {
     ja: '事前に定義された値のいずれかに等しくなければいけない',
     ko: '미리 정의된 값중 하나여야 합니다',
     nb: 'må være lik en av de forhåndsdefinerte verdiene',
-    nl: 'moet gelijk zijn aan één van de voorgedefinieerde waardes',
+    nl: 'moet overeenkomen met één van de voorgedefinieerde waarden',
```

Comment:
Although I can't think of any relevant examples where "overeenkomen" means "mostly the same", I agree that "gelijk zijn aan" would be the better answer. "overeenkomen" could be replaced by "volledig overeenkomen" but this will only make the message longer and harder to read.

"waardes" is grammatically correct, but it sounds a bit simplistic (or childish) in my opinion.